### PR TITLE
Tutorial for running TVM on Arm(R) Cortex(R)-M55 CPU and Ethos(TM)-U55 NPU

### PR DESCRIPTION
**Pull request by grant-arm**
This PR introduces a tutorial for running TVM on Arm(R) Cortex(R)-M55 CPU and Ethos(TM)-U55 NPU, using bare metal.

In order to run the application without having access to a Cortex(R)-M55 and Ethos(TM)-U55 development board, the sample application will be run on a Fixed Virtual Platform (FVP). The FVP based on Arm(R) Corstone(TM)-300 software, models a hardware system containing a Cortex(R)-M55 and Ethos(TM)-U55.

This tutorial compiles a MobileNet v1 model and instructs TVM to offload operators to the Ethos(TM)-U55 where possible.

@manupa-arm @Mousius @leandron @areusch

### CODE REVIEW

File: gallery/how_to/work_with_microtvm/micro_ethosu.py [hunk 1]
```diff
@@ -0,0 +1,1014 @@
...
+#     }
+#
+#     TVM_DLL int TVMFuncRegisterGlobal(const char* name, TVMFunctionHandle f, int override) { return 0; }
+#
+#     #ifdef __cplusplus
+#     }
+#     #endif
+
+################################################################################
+# Creating the linker script
```
Comment (areusch) [inline gallery/how_to/work_with_microtvm/micro_ethosu.py:520]: just wondering if we should include stuff like the linker script here or just link to it in the codebase from tutorial? it's a bit low level is all. on the other hand it is quite informative to understand the system.

**Comment by areusch** [comment]
@grant-arm thanks for sending this! will defer to others for review

**Comment by grant-arm** [comment]
@leandron @mbaret Would you mind having a look at this, since you have some experience developing the tvmc tutorial.

**Review by leandron — APPROVED** [review]
LGTM

**Pull request merged**